sdk/openai/openai/src/models/options.ts

```diff
@@ -233,4 +233,14 @@ export interface GetChatCompletionsOptions extends OperationOptions {
    *   This additional specification is only compatible with Azure OpenAI.
    */
   azureExtensionOptions?: AzureExtensionsOptions;
+  /**
+   * Whether to return log probabilities of the output tokens or not.If true, returns the log probabilities of
+   * each output token returned in the `content` of `message`. This option is currently not available on the `gpt-4-vision-preview` model.
```

Comment:
nitpick: we probably don't need to discuss availability of the features in the SDK docs because it may change and the customer should always consult the service docs for it.
```suggestion
   * each output token are returned in the `content` of `message`.
```